<?xml version="1.0"?>
<launch>

    <arg name="controller_name" default="tent_building_velocity_controller" />

    <arg name="control_rate" default="100"/>

    <node type="velocity_controller_single_min_dist.py" name="$(arg controller_name)" pkg="deformable_manipulations_tent_building" output="screen"> 
        <rosparam command="load" file="$(find deformable_manipulations_tent_building)/config/velocity_controller.yaml" subst_value="true" />

        <rosparam param="pub_rate_odom"                      subst_value="true">$(arg control_rate)</rosparam>
        
    </node>

</launch>
